portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path

```diff
@@ -338,6 +338,27 @@
 	<td></td>
 	<td></td>
 </tr>
+<!--PAGE-->
```

Comment:
Hi @leilanyaragao , I think that all these paths are not related to a Form Field. I believe that this path is more related to a Form feature, so please could you remove it from this file and change it to the Form.path file, please?